src/main/resources/config/tpcds/task_library.yaml

```diff
@@ -10,7 +10,7 @@ task_templates:
 - id: setup_data_maintenance
   files:
   - src/main/resources/scripts/tpcds/setup_data_maintenance/spark/ddl-external-tables-refresh.sql
-  parameter_values_file: src/main/resources/aux/tpcds/setup_data_maintenance/parameter_values.dat
+  parameter_values_file: src/main/resources/supplements/tpcds/setup_data_maintenance/parameter_values.dat
```

Comment:
Thanks for the quick PR @anjagruenheid ! Do you think 'auxiliary' is a better name?